doc/classes/OmniLight3D.xml

```diff
@@ -14,8 +14,11 @@
 	</tutorials>
 	<members>
 		<member name="omni_attenuation" type="float" setter="set_param" getter="get_param" default="1.0">
-			The light's attenuation (drop-off) curve. A number of presets are available in the [b]Inspector[/b] by right-clicking the curve. Zero and negative values are allowed but can produce unusual effects.
-			[b]Note:[/b] Very high [member omni_attenuation] values (typically above 10) can impact performance negatively if the light is made to use a larger [member omni_range] to compensate. This is because culling opportunities will become less common and shading costs will be increased (as the light will cover more pixels on screen while resulting in the same amount of brightness). To improve performance, use the lowest [member omni_attenuation] value possible for the visuals you're trying to achieve.
+			Controls the distance attenuation function for omnilights.
+			[b]Note:[/b] A value of [code]0[/code] smoothly attenuates light at the range edges. A value of [code]1[/code] approaches a physical lighting model. A value of [code]0.5[/code] approximates linear attenuation.
```

Comment:
```suggestion
			[b]Note:[/b] A value of [code]0[/code] smoothly attenuates light at the edge of the range. A value of [code]1[/code] approaches a physical lighting model. A value of [code]0.5[/code] approximates linear attenuation.
```
The edges here would be at 0 and max, as it's circular, so range singular is more correct imo